implementations/rust/ockam/ockam_command/src/node/create.rs

```diff
@@ -216,12 +223,14 @@ impl CreateCommand {
             || self.config_args.enrollment_ticket.is_some()
     }
 
-    /// Return true if the `name` argument is not a config file path or URL
+    /// Return true if the `name` argument is not a URL, a file path, or an inline config
     fn has_name_arg(&self) -> bool {
+        let is_url = is_url(&self.name).is_some();
         let is_file = std::fs::metadata(&self.name)
             .map(|m| m.is_file())
             .unwrap_or(false);
-        is_url(&self.name).is_none() && !is_file
+        let is_inline_config = serde_yaml::from_str::<NodeConfig>(&self.name).is_ok();
+        !is_url && !is_file && !is_inline_config
     }
 
     fn parse_args(&self) -> miette::Result<()> {
```

Comment:
Is `color_primary("NAME_OR_CONFIGURATION"),` on line 255 still valid?

Reply:
what do you mean? because of the arg name?

Reply:
Yes, I was wondering if the `NAME_OR_CONFIGURATION` was describing the right thing, but it does.